torch/_inductor/codegen/wrapper.py

```diff
@@ -150,6 +173,38 @@ def codegen(self, code: IndentedBuffer):
         )
 
 
+class DeferredKernelNameLine(DeferredLineBase):
+    """A line that contains mangled CUDA kernel name that will be filled in the second pass"""
+
+    def __init__(self, name, line):
+        super().__init__(line)
+        self.name = name
+
+    def __call__(self):
+        with open(f"{self.name}.mangled", "r") as f:
```

Comment:
If we need tempfile here and below (I dislike the tempfile thing):
1) We should make sure it is threadsafe if multiple PyTorch procs are running -- I don't think this is
2) We should put the tempfiles in the torchindutor cache dir not the working dir.